doc/bakugo_spec.md

```diff
@@ -546,15 +547,15 @@ The _scope_ of a declared identifier is the extent of source text in which the i
 constant, type, variable, function, label, or package.
 
 
-Go is lexically scoped using _blocks_:
+Go is lexically scoped:
 
 1. The scope of a _predeclared identifier_ is the universe block.
-2. The scope of an identifier denoting a function parameter, or result variable is the function body.
-3. The scope of a constant or variable identifier declared inside a function begins at the end of the ConstSpec or VarSpec and ends at the end of the innermost containing block.
-4. The scope of a type identifier declared inside a function begins at the identifier in the TypeSpec and ends at the end of the innermost containing block.
+1. The scope of an identifier denoting a function result variable is the function body.
```

Comment:
Does this mean that an identifier denoting a function result variable cannot be re-declared in the function body?